py3status/modules/google_calendar.py

```diff
@@ -0,0 +1,424 @@
+# -*- coding: utf-8 -*-
+"""
+Display upcoming Google Calendar events.
+
+This module will display information about upcoming Google Calendar events in
+one of two formats which can be toggled with a button press. The
+events URL may also be opened in a web browser with a button press.
+
+Configuration parameters:
+    auth_token: The path to where the access/refresh token will be saved
+        after successful credential authorization.
+        (default '~/.config/py3status/google_calendar.json')
+    button_open: Opens the URL for the clicked on event in the default
+        web browser.
+        (default 3)
+    button_toggle: The button used to toggle between output formats
+        format_event_1 and format_event_2.
+        (default 1)
+    cache_timeout: How often the module is refreshed in seconds
+        (default 60)
+    client_secret: the path to your client_secret.json file which
+        contains your OAuth 2.0 credentials.
+        (default '')
+    colors: A list of color strings specifying what color to display events
+        the first color in the list will be used for the next upcoming
+        event, the second color for the event after that, and so on; if
+        the number of colors is less than the number of events displayed,
+        the rest of the list will be padded with '#FFFFFF' (white).
+        (default [])
+    event_time_suffix:  The suffix used to indicate that the {time_to}
+        format placeholder is referring to the time until an event in progress is
+        over; e.g. if an event is 10 minutes past its start time and there are 50 minutes
+        left until it is over, '{time_to} event_in_progress_suffix' will be displayed.
+        (default 'Remaining')
+    format: The format for module output.
+        (default '{events}')
```

Comment:
This might interest you. You can use `format = '{events}|No Events'` or whatever you like to ensure that the module is still running when we don't have any events today.